velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -122,23 +123,24 @@ std::tuple<std::string, bool> WindowFuzzer::generateFrameClause() {
   // k RANGE frames require more work as we have to generate columns with the
   // frame bound values.
   std::vector<core::WindowNode::BoundType> startBoundOptions, endBoundOptions;
-  if (isRowsFrame) {
+  // if it is a range frame and NOT a k-range frame
+  if (typeOfFrame == 1) {
```

Comment:
Maybe rename this variable to `isSingleSortingKey` and decide whether to generate k-range frames based on this value? Or add an enum to represent `typeOfFrame`?